src/modules/accommodations/entities/accommodations.entity.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  Entity,
+  Column,
+  PrimaryGeneratedColumn,
+  ManyToOne,
+  CreateDateColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+import { User } from '#/modules/user/entities/user.entity';
+
+export enum AccommodationStatus {
+  AVAILABLE = 'available',
+  OCCUPIED = 'occupied',
+  NOT_AVAILABLE = 'not available',
+}
+
+@Entity('accommodations')
+export class Accommodation {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @ManyToOne(() => User, (user) => user.accommodations)
+  user: User;
+
+  @Column()
+  name: string;
+
+  @Column({ type: 'text' })
+  description: string;
+
+  @Column({ type: 'varchar', length: 255 })
+  location: string;
+
+  @Column({ type: 'decimal', precision: 10, scale: 2 })
+  pricePerNight: number;
+
```

Comment:
Please add ```name: 'price_per_night'``` in the column decorator options. Then, the name of database column will be ```price_per_night```